lib/shopify_transporter/exporters/magento/soap.rb

```diff
@@ -91,7 +91,7 @@ def soap_session_id
             }
           ).body
 
-          session_id = result&.dig(:login_response, :login_return)
+          session_id = result&.dig(:login_response, :login_return) || result&.dig(:login_response_param, :result)
```

Comment:
Wait, how do we know this is the end of it? This may solve the one problem but does each user's API have the chance of having some weird result? How many permutations of `login_response` can there be?

Reply:
Well, yeah. Like I said, this is _not_ a permanent solution, it's a stopgap.

Reply:
But are we not investigating why this sort of thing is happening and how weird it can get? Are we comfortable shipping fixes to things we're not fully understanding?

Reply:
100%. Like I said in the issues, I want to investigate more deeply into why this is actually happening. If it was just one user, I'd tell them to fork the code and fix it on their end while we do that. But since 2 separate users already logged the same problem, I'm assuming it's widespread and I'd rather merge a fix while we continue investigating.

Reply:
👍